.env.example

```diff
@@ -0,0 +1,3 @@
+NODE_ENV=development
```

Comment:
.env file should be part of .gitignore, please remove this file

Reply:
It is a .example file, and its intention is to help the interns to know all the variables available in the system, they will copy this file to create their own .env file